tests/wpunit/Tribe/functions/templateTags/postTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Tribe\functions\templateTags;
+
+class postTest extends \Codeception\TestCase\WPTestCase {
+
+	public function setUp() {
+		// before
+		parent::setUp();
+
+		// your set up methods here
+	}
+
+	public function tearDown() {
+		// your tear down methods here
+
+		// then
+		parent::tearDown();
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_does_not_run_apply_filters_when_doing_filter_and_create_nested_level_error() {
+		// Remove autop to make it easier to match content.
+		remove_filter( 'the_content', 'wpautop' );
+		$content = 'Test Content';
+		$post_id = $this->factory->post->create();
+		$expected = $content . tribe_get_the_content( null, false, $post_id ) . 'FILTERING';
```

Comment:
This is not readable and brittle.
If the `tribe_get_the_content` function does not work correctly, then we'll a bad expectation yielding either false positives or false negatives.

Refactor this to be a simple, readable, string of what we expect to get, e.g. `Test Content FILTERING`, and use that for comparison.

"Smart" expectations are destined to break, I speak from experience.